root/report/components/EditorList.js

```diff
@@ -0,0 +1,57 @@
+/*
+ * @flow
+ * Copyright (C) 2018 MetaBrainz Foundation
+ *
+ * This file is part of MusicBrainz, the open internet music database,
+ * and is licensed under the GPL version 2, or (at your option) any
+ * later version: http://www.gnu.org/licenses/gpl-2.0.txt
+ */
+
+import * as React from 'react';
+
+import {l} from '../../static/scripts/common/i18n';
+import PaginatedResults from '../../components/PaginatedResults';
+import EditorLink from '../../static/scripts/common/components/EditorLink';
+import loopParity from '../../utility/loopParity';
+import type {ReportEditorT} from '../types';
+
+const EditorList = ({
+  items,
+  pager,
+}: {items: $ReadOnlyArray<ReportEditorT>, pager: PagerT}) => (
+  <PaginatedResults pager={pager}>
+    <table className="tbl">
+      <thead>
+        <tr>
+          <th>{l('Editor')}</th>
+          <th>{l('Member since')}</th>
+          <th>{l('Website')}</th>
+          <th>{l('Email')}</th>
+          <th>{l('Bio')}</th>
+        </tr>
+      </thead>
+      <tbody>
+        {items.map((item, index) => {
+          { /* Hack to make this linkable with EditorLink */ }
+          { /* $FlowFixMe */ }
+          item.entityType = 'editor';
```

Comment:
There should be a simple way we can clean this all up: in lib/MusicBrainz/Server/Report/LimitedEditors.pm, just add an `inflate_rows` method that creates an `Entity::Editor` from each row. Right now it's unfortunate how this `ReportEditorT` is leaking to other parts of the codebase (EntityLink, entityHref) despite only being invented for one file. :/

Reply:
Changed this to make it less hacky, and to load Editor hopefully it looks good now!